.travis.yml

```diff
@@ -1,8 +1,6 @@
 language: php
-
-php:
-  - 7.1
-  - 7.2
+php: [7.1, 7.2, nightly ]
+sudo: false
 
 matrix:
```

Comment:
Unsure if leaving this `matrix` section in here actually works with the newer file format.  Build output doesn't appear to do the separate ECS and PHPSTAN builds that were done via the old format.